README.md

```diff
@@ -63,7 +63,14 @@ This project is licensed under the MIT License.
 
 ## Running in docker containers
 
-It is not possible to pull from the ecr repos if working locally, you must build the images from source
+It is not possible to pull from the ecr repos if working locally, you must build the images from source. Inside the root of each repo, run `docker build -t <some-image-name> .` Then specify those images are to be used by setting env vars as follows:
```

Comment:
You shouldn't have to do this as this is what happens in `docker-compose.local.yml`. Can we check before adding this in? I think it might be version of docker desktop or docker engine.

Reply:
Maybe we can leave this in but worded as an alternative if using the local override doesn't work?

Reply:
Will remove this - you're right, my issue was out-of-date docker. Will fix.